### DISCUSSION TITLE
Permission to claim more blocks

### DISCUSSION THREAD

**Discussion by ax3lt** [GriefPrevention/GriefPrevention#1117] [Q&A] [answered]
Is there permission to add blocks to the default ones? I tried to search a bit but I can't find anything
In practice I would need for example that the VIPs have 10k more blocks to claim

**Comment by Jikoo** [comment, marked as answer]
Not sure if you found what you were looking for or not, but for future people with the same question, [GPAccruals](https://github.com/TechFortress/GriefPrevention/wiki/Addons#gpaccruals) should add what you need.
